src/Field.tsx

```diff
@@ -78,6 +81,17 @@ export interface FieldConfig {
 
   /** Inner ref */
   innerRef?: (instance: any) => void;
+
+  shouldFieldUpdateOrEnableDefault?:
```

Comment:
how about just `shouldUpdate`?

Reply:
en, `shouldFieldUpdateOrEnableDefault` may be boolean type, and indicates whether built-in logic is enabled.